modules/style.js

```diff
@@ -23,6 +24,19 @@ function updateStyle(oldVnode, vnode) {
       elm.style[name] = cur;
     }
   }
+
+  for (name in oldStyle) {
+    if (name === 'delayed') {
+      for (name in oldStyle.delayed) {
+        if (!curHasDel || !(name in style.delayed)) {
+          elm.style[name] = '';
+        }
+      }
+    }
+    if (name !== 'remove' && name !== 'destroy' && !(name in style)) {
+      elm.style[name] = '';  // the DOM is weird eh?
```

Comment:
Doesn't this mean that instead of supporting remove users can just set CSS values to the empty string?